default.nix

```diff
@@ -99,45 +99,40 @@ rec {
   ];
 
   # sources for integration tests
-  tests = pkgs.runCommandLocal "tests"
-    {
-      src = lib.sourceByRegex ./. [
-        "^integration_tests$"
-        "^integration_tests/.*\\.py$"
-        "^integration_tests/configs$"
-        "^integration_tests/configs/.*"
-      ];
-    } ''
-    mkdir -p $out/share
-    ln -s $src/integration_tests $out/share/
-  '';
+  tests_src = lib.sourceByRegex ./. [
+    "^integration_tests$"
+    "^integration_tests/.*\\.py$"
+    "^integration_tests/configs$"
+    "^integration_tests/configs/.*"
+  ];
 
   # an env which can run integration tests
-  ci-env-inner = pkgs.buildEnv {
-    name = "ci-env-inner";
+  ci-env = pkgs.buildEnv {
+    name = "ci-env";
     pathsToLink = [ "/bin" "/share" ];
     paths = with pkgs; [
       lint-env
       chain-maind-zemu
       chain-maind-zemu.instrumented
-      tests
     ] ++ common-env;
   };
 
   # main entrypoint script to run integration tests
   run-integration-tests = pkgs.writeShellScriptBin "run-integration-tests" ''
-    export PATH=${ci-env-inner}/bin:$PATH
-    export TESTS=${ci-env-inner}/share/integration_tests
+    export PATH=${ci-env}/bin:$PATH
```

Comment:
I think we need a “set -e” here, to stop on error.

Reply:
yes, it's reporting now! thanks